pubnub/request_handlers/requests_handler.py

```diff
@@ -32,10 +32,10 @@ class RequestsRequestHandler(BaseRequestHandler):
     def __init__(self, pubnub):
         self.session = Session()
 
-        self.session.mount('http://ps.pndsn.com', HTTPAdapter(max_retries=1, pool_maxsize=500))
-        self.session.mount('https://ps.pndsn.com', HTTPAdapter(max_retries=1, pool_maxsize=500))
-        self.session.mount('http://ps.pndsn.com/v2/subscribe', HTTPAdapter(pool_maxsize=500))
-        self.session.mount('https://ps.pndsn.com/v2/subscribe', HTTPAdapter(pool_maxsize=500))
+        self.session.mount('http://%s' % pubnub.config.origin, HTTPAdapter(max_retries=1, pool_maxsize=500))
+        self.session.mount('https://%s' % pubnub.config.origin, HTTPAdapter(max_retries=1, pool_maxsize=500))
+        self.session.mount('http://%s/v2/subscribe' % pubnub.config.origin, HTTPAdapter(max_retries=1, pool_maxsize=500))
+        self.session.mount('https://%s/v2/subscribe' % pubnub.config.origin, HTTPAdapter(max_retries=1, pool_maxsize=500))
```

Comment:
The subscribe paths were originally mounted separately so they do not get the same retry behavior as the default pools. If you want the same retry behavior on subscribes, these last two lines can just be deleted.

Reply:
Figured it allowed separate dedicated pool - based on docs - https://requests.readthedocs.io/en/master/api/#requests.adapters.HTTPAdapter

The mount of a new adapter will create dedicated pool.  I think?

`max_retries` only refers to TCP/TLS sockets. It does not retry on HTTP.  That can be set in another config, however may be fun to add that at some point?

Reply:
lmk if this doesn't create dedicated pool, and would be good to simplify as you suggest :+1:

Reply:
It does create separate pools, but I'm not sure why that would be useful.

It looks like the original intent was for the general pool to have `max_retries=1` and the subscribe pool to just have the default. I also do not have context for why this would be wanted. But that's what it did.

Reply:
best to play safe, I'll set this to original design intent to keep changes to minimal :+1:

Reply:
restored 🙌